leptos_macro/src/view.rs

```diff
@@ -484,42 +489,35 @@ fn element_to_tokens_ssr(
                             );
                         }
                         Node::Text(text) => {
-                            if let Some(value) = value_to_string(&text.value) {
-                                let value = if is_script_or_style {
-                                    value.into()
-                                } else {
-                                    html_escape::encode_safe(&value)
-                                };
-                                template.push_str(
-                                    &value
-                                        .replace('{', "\\{")
-                                        .replace('}', "\\}"),
-                                );
+                            let value = text.value_string() ;
+                            let value = if is_script_or_style {
+                                value.into()
                             } else {
-                                template.push_str("{}");
-                                let value = text.value.as_ref();
-
-                                holes.push(quote! {
-                                  #value.into_view(#cx).render_to_string(#cx)
-                                })
-                            }
+                                html_escape::encode_safe(&value)
+                            };
+                            template.push_str(
+                                &value
+                                    .replace('{', "\\{")
+                                    .replace('}', "\\}"),
+                            );
                         }
                         Node::Block(block) => {
-                            if let Some(value) = value_to_string(&block.value) {
-                                template.push_str(&value);
-                            } else {
-                                let value = block.value.as_ref();
-
-                                if !template.is_empty() {
-                                    chunks.push(SsrElementChunks::String {
-                                        template: std::mem::take(template),
-                                        holes: std::mem::take(holes),
-                                    })
-                                }
-                                chunks.push(SsrElementChunks::View(quote! {
-                                  {#value}.into_view(#cx)
-                                }));
+                            // TODO: Is it was string? Original syn-rsx parse block with braces,
+                            // but value_to_string expect it without braces.
```

Comment:
If I understand you correctly: `{"Text"}` should be handled as `SsrElementChunks::String` here, I think this was just a mistake in my origin `value_to_string`, which you've corrected.